use python3 binary in the helper script because not all users will have set a
python symlink

Index: script/homm2/extract_homm2_resources.sh
--- script/homm2/extract_homm2_resources.sh.orig
+++ script/homm2/extract_homm2_resources.sh
@@ -82,8 +82,8 @@ if [[ ! -f "$HOMM2_PATH/homm2.gog" ]]; then
 fi
 
 # Special case - CD image from GOG
-if [[ "$(command -v python)" == "" ]]; then
-    echo_yellow "python was not found in your system. Please install it and re-run this script to extract animation resources."
+if [[ "$(command -v python3)" == "" ]]; then
+    echo_yellow "python3 was not found in your system. Please install it and re-run this script to extract animation resources."
     exit 0
 fi
 if [[ "$(command -v bsdtar)" == "" ]]; then
@@ -93,7 +93,7 @@ fi
 
 echo_green "Extracting animation resources, please wait..."
 
-python - << EOF
+python3 - << EOF
 with open("$HOMM2_PATH/homm2.gog", "rb") as raw_file:
     with open("homm2.iso", "wb") as iso_file:
         while True:
